### PR TITLE
*: Add a variable to skip constraint check in session.

### PR DESCRIPTION
**Pull request by shenli**
When import data from mysqldump/mydumper. We could skip constraint check.
Test cases will be added in the next commit.

### CODE REVIEW

File: sessionctx/variable/session.go [hunk 1]
```diff
@@ -291,6 +296,12 @@ func (s *SessionVars) setSnapshotTS(sVal string) error {
 	return nil
 }
 
+func (s *SessionVars) setSkipConstraintCheck(sVal string) {
+	if sVal == "1" {
+		s.SkipConstraintCheck = true
+	}
```
Comment (coocood) [inline sessionctx/variable/session.go]: else set false.

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**